packages/contentstack-import/src/import/modules/entries.ts

```diff
@@ -56,6 +56,8 @@ export default class EntriesImport extends BaseClass {
   private autoCreatedEntries: Record<string, any>[];
   private taxonomiesPath: string;
   public taxonomies: Record<string, unknown>;
+  public rteCTs: any;
+  public rteCTsWithRef: any;
```

Comment:
@cs-raj private will be fine here?

Reply:
While adding the test cases, It will be directly accessible